coverage/coverage-final.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Just curious, what is the purpose of this file?

Reply:
oh, this is automatically generated file from jest, I will put it into git ignore.